packages/terminal/src/browser/base/terminal-widget.ts

```diff
@@ -178,9 +178,9 @@ export interface TerminalWidgetOptions {
     readonly title?: string;
 
     /**
-     * icon class
+     * icon class with or without color modifier
      */
-    readonly iconClass?: string;
+    readonly iconClass?: string | {icon: string, color: string};
```

Comment:
Suggestion: Since this is a breaking change anyway, would it make sense to type this differently? This would make dealing with this value much more straightforward. We should also preferably export the underlying type.

```ts
export interface TerminalIcon {
    icon: string;
    color?: string;
}
```

```suggestion
    readonly iconClass?: TerminalIcon;
```

Reply:
I like this and it should be an easy enough implementation.